gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.13-bin.zip
```

Use 8.11.1 version, re-generate the Gradle wrapper and commit it.

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.11.1-bin.zip
```